instaloader/structures.py

```diff
@@ -936,6 +936,11 @@ def followers(self) -> int:
     def followees(self) -> int:
         return self._metadata('edge_follow', 'count')
 
+    @property
+    def bio_links(self) -> List[str]:
+        """list of external links given in the bio"""
+        return [links['url'] for links in self._metadata('bio_links')] if self._metadata('bio_links') else []
```

Comment:
I do not understand the purpose of `if self._metadata('bio_links')` here. If it evaluates to false due to an empty list, the left-hand statement would iterate an empty list and return an empty list, without that it needs to be explicitly checked.